dali/test/python/test_optical_flow.py

```diff
@@ -23,13 +23,14 @@ def __init__(self, batch_size, num_threads, device_id):
     def define_graph(self):
         seq = self.input(name="Reader")
         of = self.of_op(seq.gpu())
-        return of,seq.gpu()
+        return of
 
 
 def test_of():
     pipe = OFPipeline(batch_size=batch_size, num_threads=1, device_id=0)
     pipe.build()
     pipe_out = pipe.run()
     frames = pipe_out[0].as_cpu().as_array()
-    myarray=np.loadtxt(image_dir+'decoded_flow_vector.dat')
-    assert (0.9 < np.mean(np.abs(frames[0][0].flatten()-myarray)))
+    myarray = np.loadtxt(image_dir + '/../decoded_flow_vector.dat')
```

Comment:
Maybe to be consistent, another path created with:
`os.path.join(dali_extra_path, 'db', 'optical_flow', 'slow_preset', 'decoded_flow_vector.dat')`
With it we could get rid of this `/../`

Reply:
done